mux.go

```diff
@@ -210,7 +210,7 @@ func (m *mux) DoMultiStream(ctx context.Context, multi ...Completed) MultiRedisR
 }
 
 func (m *mux) Do(ctx context.Context, cmd Completed) (resp RedisResult) {
-	if m.usePool && !cmd.NoReply() {
+	if m.usePool && !cmd.NoReply() && !cmd.IsPipe() {
```

Comment:
```suggestion
	if m.usePool && !cmd.IsPipe() {
```
Can we just keep IsPipe() here? That is, add the new pipeTag into the existing noRetTag.